pubmed_crawler.py

```diff
@@ -300,15 +300,20 @@ def find_publications(syn, grant_id, table_id, email):
     """
     grants = get_grants(syn, grant_id)
     pmids = get_pmids(grants)
+    
 
     # If user provided a table ID, only scrape info from publications
     # not already listed in the provided table.
     if table_id:
         table_name = syn.get(table_id).name
+        if table_id == "syn52752398":
+            id_col = "Pubmed Id"
+        else:
+            id_col = "pubMedId"
```

Comment:
I'm a big fan of using the ternary operator for situations like these where there are only two possible values based on a single conditional:

```suggestion
        id_col = "Pubmed Id" if table_id == "syn52752398" else "pubMedId"
```

Reply:
Nice! Yeah, that looks better to me, too! Thank you!